mmv1/third_party/terraform/utils/iam.go.erb

```diff
@@ -242,6 +242,36 @@ func subtractFromBindings(bindings []*cloudresourcemanager.Binding, toRemove ...
 	return listFromIamBindingMap(currMap)
 }
 
+func iamMemberIsCaseSensitive(member string) bool {
+	return strings.HasPrefix(member, "principalSet:") || strings.HasPrefix(member, "principal:") || strings.HasPrefix(member, "principalHierarchy:")
+}
+
+func normalizeIamMemberCasing(member string) string {
+	// members are in <type>:<value> format
```

Comment:
Can you update this comment based on the changes to the code?

Reply:
We'll probably also want to make it a function comment instead of an inline one.